streams.yml

```
@@ -39,6 +39,19 @@ golang:
   # tests that don't happen downstream.
   upstream_image: registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.17-openshift-{MAJOR}.{MINOR}
 
+golang-1.18:
+  image: openshift/golang-builder@sha256:<wip>
```

updating with built image
```suggestion
  # openshift-golang-builder-container-v1.18.1-1.scratch.nofips.g04fb1ce - an experimental build, YMMV
  image: openshift/golang-builder@sha256:0f4c69877fe9e6b87ab0d6f70085d16f1fb1d523bf0aaa17cf3f84a7849fcc82
```